docs/_navbar.md

```diff
@@ -0,0 +1,5 @@
+* <i class="fab fa-discord"></i> [Discord]((https://discord.gg/3C4qeG8XhE)
```

Comment:
```suggestion
* <i class="fab fa-discord"></i> [Discord](https://discord.gg/3C4qeG8XhE)
```
The discord link seems not to render properly; this may be why